.github/workflows/ci.yml

```diff
@@ -7,129 +8,123 @@ on:
       - releases/*
   pull_request: null
 
+env:
+  CACHE_NUMBER: 0  # increase to reset cache manually
+  CONDA_ENV: .github/environment.yml
```

Comment:
what does this do?

Reply:
The block sets environment variables.
`CACHE_NUMBER` is used in the cache key, and if the developer wants to trigger cache reset manually they only need to change this number for the cache key comparison to fail, and the packages to be re-downloaded.
`CONDA_ENV` is a file with the conda environment packages. I took them out of the workflow to declutter.